docs/environments/third_party_environments.md

```diff
@@ -285,7 +292,6 @@ goal-RL ([Gymnasium-Robotics](https://robotics.farama.org/)).
 
   SimpleGrid is a super simple and minimal grid environment for Gymnasium. It is easy to use and customise and it is intended to offer an environment for rapidly testing and prototyping different RL algorithms.
 
-
```

Comment:
why was this line removed?

Reply:
I accidentally cut it when I cut-pasted the description of my environment, sorry. Reverted.